### DISCUSSION TITLE
[zksync-cli] <Title> git clone problem

### DISCUSSION THREAD

**Discussion by 0xliu1shou** [zkSync-Community-Hub/zksync-developers#404] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement.

I'm trying to set up a project via command "npx zksync-cli create", but problem happened

### What is the specific issue or error you're encountering?

can't create project with zksync-cli

### Can you share the error messages or logs you're receiving, if any?

fatal: Too many arguments

### Have you made any recent changes to the contract before encountering this issue?

no

### Are there any external libraries or contracts that your contract interacts with?

no

### Can you provide the relevant portions of your contract code where the issue is occurring?

no

### Have you tried to isolate the problem, and if so, what were the results?

no

### What steps have you already taken to try to resolve the issue?

I use Mac Os, IDE is VS Code, node.js installed, git version 2.44.0, zksync-cli version 1.5.1

**Comment by JackHamer09** [comment, marked as answer]
Hey! Most likely this happens because of space in the path where you're trying to run `create` script. However most popular terminals should handle such cases automatically.
I recommend trying to update your `git` and terminal (e.g., [git bash](https://git-scm.com/downloads)). 
Or simply try to use a different path (so that there are no spaces in the path) or a different terminal.
